packages/common/src/components/CondaPkgList.tsx

```diff
@@ -96,36 +90,36 @@ export class CondaPkgList extends React.Component<IPkgListProps> {
       if (pkg.version_selected === 'none') {
         return (
           <FontAwesomeIcon
-            icon={faMinusSquare as IconProp}
+            icon="minus-square"
             style={{ color: 'var(--jp-error-color1)' }}
           />
         );
       } else if (pkg.version_selected !== pkg.version_installed) {
         return (
           <FontAwesomeIcon
-            icon={faExternalLinkSquareAlt as IconProp}
+            icon="external-link-square-alt"
             style={{ color: 'var(--jp-accent-color1)' }}
           />
         );
       }
       return (
         <FontAwesomeIcon
-          icon={faCheckSquare as IconProp}
+          icon="check-square"
           style={{ color: 'var(--jp-brand-color1)' }}
         />
       );
     } else if (pkg.version_selected !== 'none') {
       return (
         <FontAwesomeIcon
-          icon={faCheckSquare as IconProp}
-          style={{ color: 'var(--jp-brand-color1)' }}
+          icon="square"
+          style={{ color: 'var(--jp-ui-font-color1)' }}
```

Comment:
Was this color name changed in the latest version?

Reply:
This one is also from tab complete! Thank you, I will update this one!